modules/local/cat.nf

```diff
@@ -11,14 +11,14 @@ process CAT {
     tuple val(db_name), path("database/*"), path("taxonomy/*")
 
     output:
-    path("*.ORF2LCA.names.txt.gz")            , emit: orf2lca_classification
-    path("*.bin2classification.names.txt.gz") , emit: tax_classification_names
-    path("raw/*.ORF2LCA.txt.gz")              , emit: orf2lca
-    path("raw/*.predicted_proteins.faa.gz")   , emit: faa
-    path("raw/*.predicted_proteins.gff.gz")   , emit: gff
-    path("raw/*.log")                         , emit: log
-    path("raw/*.bin2classification.txt.gz")   , emit: tax_classification_taxids
-    path "versions.yml"                       , emit: versions
+    tuple val(meta), path("*.bin2classification.names.txt")    , emit: tax_classification_names
+    path("*.ORF2LCA.names.txt.gz")                             , emit: orf2lca_classification
+    path("raw/*.ORF2LCA.txt.gz")                               , emit: orf2lca
+    path("raw/*.predicted_proteins.faa.gz")                    , emit: faa
+    path("raw/*.predicted_proteins.gff.gz")                    , emit: gff
+    path("raw/*.log")                                          , emit: log
+    path("raw/*.bin2classification.txt.gz")                    , emit: tax_classification_taxids
+    path "versions.yml"                                        , emit: versions
```

Comment:
Do any of the res tof these files also need a meta? e.g. the names? Or are they not used for anything else?

Reply:
None of these files are re-used later, hence no meta for them.